examples/demo_coordinate_system_datasets.ipynb

```diff
@@ -0,0 +1,916 @@
+    "ax.set_ylabel(r\"$g_t$\", fontsize=10)\n",
+    "ax.grid(lw=0.3)\n",
+    "ax.minorticks_on()\n",
+    "ax.grid(which=\"minor\", lw=0.1)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we create a `GalaxyCluster` object, specifying the *correct* coordinate system.  For source galaxies from CosmoDC2, these are in the **euclidean** coordinate system.  We use the implemented kwarg when defining the galaxy cluster object to also specify the **euclidean** coordinate system.  However, with a single galaxy cluster, the signal is not significant enough to clearly see a difference.  We do notice an excess with the correct coordinate system at larger radii."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cluster2 = clmm.GalaxyCluster(\"Name of cluster\", cluster_ra, cluster_dec, cluster_z, galaxies, coordinate_system=\"euclidean\")\n",
+    "cluster2.compute_tangential_and_cross_components()\n",
+    "print(cluster.galcat.colnames)\n",
+    "\n",
+    "# Measure profile and add profile table to the cluster.\n",
+    "seps = convert_units(cluster2.galcat[\"theta\"], \"radians\", \"Mpc\", cluster2.z, cosmo)\n",
+    "\n",
+    "cluster2.make_radial_profile(\n",
+    "    bins=da.make_bins(0.1, 3.0, 15, method=\"evenlog10width\"),\n",
+    "    bin_units=\"Mpc\",\n",
+    "    cosmo=cosmo,\n",
+    "    include_empty_bins=False,\n",
+    "    gal_ids_in_bins=True,\n",
+    ")\n",
+    "print(cluster.profile.colnames)\n",
+    "fig = plt.figure(figsize=(6, 4))\n",
+    "ax = fig.add_axes((0, 0, 1, 1))\n",
+    "errorbar_kwargs = dict(linestyle=\"\", marker=\"o\", markersize=1, elinewidth=0.5, capthick=0.5)\n",
+    "ax.errorbar(\n",
+    "    cluster.profile[\"radius\"],\n",
+    "    cluster.profile[\"gt\"],\n",
+    "    cluster.profile[\"gt_err\"],\n",
+    "    label='celestial'   \n",
+    ")\n",
+    "ax.errorbar(\n",
+    "    cluster2.profile[\"radius\"],\n",
+    "    cluster2.profile[\"gt\"],\n",
+    "    cluster2.profile[\"gt_err\"],\n",
+    "    label='euclidean'\n",
+    ")\n",
+    "ax.set_xlabel(\"r [Mpc]\", fontsize=20)\n",
+    "ax.set_ylabel(r\"$g_t$\", fontsize=20)\n",
+    "ax.grid(lw=0.3)\n",
+    "ax.minorticks_on()\n",
+    "ax.grid(which=\"minor\", lw=0.1)\n",
+    "plt.legend(fontsize=20)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example source galaxies from M. Oguri\n",
+    "\n",
+    "This dataset is a curated selection of cluster and source catalogs from Summer School lectures delivered by Masamune Oguri.  There are eight galaxy clusters in this selection.  \n",
+    "\n",
+    "More details on the corresponding tutorial can be found at this [GitHub link](https://github.com/oguri/wlcluster_tutorial). The coordinate system is euclidean/pixel.  These are also in the **euclidean** coordinate system."
```

Comment:
Remove 'The coordinate system is euclidean/pixel'

Reply:
Done